tests/python/unittest/test_hybrid_script.py

```diff
@@ -2,6 +2,45 @@
 from tvm.hybrid import script
 from tvm.hybrid.intrin import HYBRID_GLOBALS
 
+def run_and_test(func, args, outs, var_dict={}, target='llvm'):
```

Comment:
avoid use xxx_test function in testcases(this will automatically be ran by nose). change to run_and_check

Reply:
I know. I am working on it. I have decorated it.